tests/test_google_lifesciences.py

```diff
@@ -90,3 +90,22 @@ def test_touch_remote_prefix():
         )
     finally:
         cleanup_google_storage(storage_prefix, bucket_name)
+
+
+@google_credentials
+def test_cloud_checkpoints_issue574():
+    """see Github issue #574"""
+    bucket_name = "snakemake-testing-%s" % next(tempfile._get_candidate_names())
+    create_google_storage(bucket_name)
+    storage_prefix = "test_cloud_checkpoints_issue574"
+    workdir = dpath("test_cloud_checkpoints_issue574")
+    try:
+        run(
+            workdir,
+            use_conda=True,
+            default_remote_prefix="%s/%s" % (bucket_name, storage_prefix),
+            google_lifesciences=True,
+            google_lifesciences_cache=True,
```

Comment:
ah, I think this should be changed based on the other functions in this file? That might be a mistake on my part.
```suggestion
            google_lifesciences_cache=False,
```

Reply:
Yes, this should be False indeed.